fieldservice/wizard/fsm_wizard.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class FSMWizard(models.TransientModel):
+    """
+        A wizard to convert a res.partner record to a fsm.person or
+         fsm.location
+    """
+    _name = 'fsm.wizard'
+    _description = 'FSM Record Conversion'
+
+    fsm_record_type = fields.Selection([('person', 'Worker'),
+                                        ('location', 'Location')],
+                                       'Record Type')
+
+    @api.multi
+    def action_convert(self):
+        partners = self.env['res.partner'].browse(
+            self._context.get('active_ids', []))
+        for partner in partners:
+            if self.fsm_record_type == 'person':
+                self.action_convert_person(partner)
+            if self.fsm_record_type == 'location':
+                self.action_convert_location(partner)
+        return {'type': 'ir.actions.act_window_close'}
+
+    def action_convert_location(self, partner):
+        pass
```

Comment:
@murtuzasaleh Even if stock is not installed, we still need to convert the partner record into a fsm.location.